scripts/parser.js

```diff
@@ -0,0 +1,71 @@
+const jobsModel = require('../app/models/jobsModel.js');
+const MBO = require('mindbody-sdk');
+const crypto = require('crypto');
+ 
+const mbo = new MBO({
+    ApiKey: process.env.API_KEY, // from portal
+    SiteId: parseInt(process.env.SITE_ID) //thisis the sandbox account
+});
+
+/**
+ * Iterates through provided class information from Mindbody API
+ * and updates or inserts jobs to the database
+ */
+async function processResponse(err,data){
+    if (err) {
+        console.error("Failed to retrieve information about classes", err);
+    } else {
+        const classes = data.Classes;
+        const presentDate = new Date();
+        presentDate.setMinutes(presentDate.getMinutes() + 15);
+        const tomorrowDate = new Date();
+        tomorrowDate.setDate(tomorrowDate.getDate() + 1);
+        for(let i = 0; i < classes.length; i++) {
+            const class_json = classes[i];
```

Comment:
I feel like this is what you are talking about, but you can still use a map function because it is just like a for each loop. The reason we want to get in the habbit of using map functions is because for loops in async functions will be skipped, and using map function, we can force async functions to wait for that particular job to finish running